features/config/upgrade/src/main/resources/changelog-cm/29.0.0/changelog-cm.xml

```diff
@@ -9,6 +9,7 @@
 
     <changeSet author="pschweizer" id="1.0-register-schema-provisiond">
         <cm:registerSchema id="provisiond" version="1.0.0" entityClass="org.opennms.netmgt.config.provisiond.ProvisiondConfiguration"/>
```

Comment:
This may be out of scope of this PR, but will mention here since it comes to mind:

The entity class, or the source used for the schema should be "frozen" for this to work reliably.

Assume an instance is upgraded from H28 to H30 and this is run for the time there.
Now assume we make changes to the XSD/model in H31.

Now, if another instance is upgraded directly from H28 to H31 - schema version 1.0.0 will be the model from H31 and not H30.

Is this being considered in the current approach?

Reply:
as discussed we will take this into a separate discussion / workstream